src/Serilog.Sinks.Console/ConsoleAuditLoggerConfigurationExtensions.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Copyright 2017 Serilog Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Serilog.Configuration;
+using Serilog.Core;
+using Serilog.Events;
+using Serilog.Formatting;
+using Serilog.Sinks.SystemConsole;
+using Serilog.Sinks.SystemConsole.Output;
+using Serilog.Sinks.SystemConsole.Themes;
+using System;
+
+namespace Serilog
+{
+    /// <summary>
+    /// Adds the AuditTo.Console() extension method to <see cref="LoggerAuditSinkConfiguration"/>.
+    /// </summary>
+    public static class ConsoleAuditLoggerConfigurationExtensions
+    {
+        static readonly object DefaultSyncRoot = new object();
+        const string DefaultConsoleOutputTemplate = "[{Timestamp:HH:mm:ss} {Level:u3}] {Message:lj}{NewLine}{Exception}";
```

Comment:
May be best to share this constant with the non-auditing version - no reason that they would diverge, AFAIK.

Reply:
done